io.mpv.Mpv.yml

```diff
@@ -725,20 +727,85 @@ modules:
 #              type: git
 #              tag-pattern: ^(\d{1,2}\.\d{1,2}\.\d{1,4})$
 
+  - name: wayland-protocols
```

Comment:
are you sure this is needed? same version is in [freedesktop](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/release/24.08/elements/components/wayland-protocols.bst#L18) runtime